packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -139,6 +139,11 @@ export enum TelemetryEvent {
   MigrateTeamsTabAppCode = "migrate-teams-tab-app-code",
   MigrateTeamsManifestStart = "migrate-teams-manifest-start",
   MigrateTeamsManifest = "migrate-teams-manifest",
+
+  TreeViewLocalDebug = "treeview-localdebug",
```

Comment:
Can we use existing RunIconDebug event with triggerFrom property to distinguish?

Reply:
They are different logic. One can choose to use local / remote debug, another only can choose local debug.